docs/model_structure.rst

```diff
@@ -10,43 +10,49 @@ Model structure
     :backlinks: top
 
 
-An `oemof.solph.EnergySystem` in the form of a tabular Datapackage form comprises scalar data
-(elements) and data with a time index (sequences). The model oemof-B3 uses
-`oemoflex <https://github.com/rl-institut/oemoflex>`_ to build empty
-DataPackages and fills them with concrete numbers.
+The input data consists of scalar data (elements) and data with a time index (sequences).
+The model oemof-B3 uses `oemoflex <https://github.com/rl-institut/oemoflex>`_ to build empty
+DataPackages and fills them with concrete numbers using preprocessed resources (see :ref:`buid_datapckage<build_datapackage>`).
 
+oemof-B3 resources
+------------------
 
-Elements
---------
+The resources, i.e. preprocessed data that serves as material for building scenarios, are prepared
+by several scripts. They follow a common data schema defined in :file:`oemof_b3/schema/`.
 
-The element-files describe the busses and components of the energy system. All `oemof.solph.Bus` es
-are defined in a single file :file:`bus.csv`. The other components are split between several files.
+Scalars
 
-Filenames for the components are of the form
-carrier-tech.csv (e.g. :file:`electricity-demand.csv`, :file:`gas-bpchp.csv`).
+.. csv-table::
+   :delim: ;
+   :file: ../oemof_b3/schema/scalars.csv
 
-There is a set of general variables that appear in all components:
+Time series
 
-* **region** Region of a component. (the regions are defined in the topology file
-  :file: ../oemof_b3/model/topology.yml)
-* **name** Unique name of the form :py:attr:`'region-carrier-tech'` (e.g. :py:attr:`'LU-gas-bpchp'`,
-  :py:attr:`'AT-electricity-airsourcehp'`).
-* **carrier** Energy carrier of the flow into the component (e.g. solar, wind, biomass, coal,
-  lignite, uranium, oil, gas, methane, hydro, waste, electricity, heat). This allows to categorize
-  the component and should correspond to the bus to which the component is connected.
-* **tech** More detailled specification of the technology (e.g. st, ocgt, ccgt, pv, onshore,
-  offshore, ror, phs,
-  extchp, bpchp, battery)
-* **type** Type of oemof.tabular.facade, defined in `TYPEMAP`.
+.. csv-table::
+   :delim: ;
+   :file: ../oemof_b3/schema/timeseries.csv
+
+A few more conventions are important to know. Missing data is left empty. If a value applies to all
+regions, this is indicated by :attr:`ALL`. If it applies to the sum of regions, by :attr:`TOTAL`.
```

Comment:
Maybe add that we do no unit transformation (right, @jnnr?), which means that e.g. all capacities have to be given in the same unit, e.g. MW.